src/blog/2024/09/node-red-version-control-with-snapshots.md

```diff
@@ -0,0 +1,238 @@
+
+### Disabling Auto Snapshots
+
+1. Switch to the **Developer Mode** tab by clicking on the "Developer Mode" option at the top.
+
+![Image showing option to enable/disable auto-snapshots for devices](./images/enable-disable-auto-snapshot.png){data-zoomable}
+*Image showing option to enable/disable auto-snapshots for devices*
+
+2. Once inside the Developer Mode tab, you will find the option to enable or disable the Auto Snapshot feature.
+
+**Note:** A limit of 10 auto snapshots is maintained, with the oldest one being deleted when a new one is created. Also this feature is only available to **Team** and **Enterprise** users.
+
+## Setting a Device Target Snapshot
+
+When you set a device target snapshot, all devices associated with the instance will be restarted and updated to run that specified snapshot. This gives you full control over when devices receive the latest changes from your development instance.
+
+With this feature, you can continue developing and testing your flows in FlowFuse without immediately impacting your devices. Once you're confident the flow is ready, you can push the changes to the devices by setting the target snapshot.
+
+Device target snapshots can be assigned either during the creation of a snapshot or at a later time.
+
+### To set the already creted Snapshot as a Device Target Snapshot:
+
+1. Go to the **Snapshots** tab and click on the three-dot icon next to the desired snapshot.
+
+![Image showing the 'Set as Device Target' option in the menu](./images/set-targe-snapshot-option.png){data-zoomable}
+*Image showing the 'Set as Device Target' option in the menu*
+
+2. Select **Set as Device Target** from the menu, prompt will open to conform then click on to the "Set Target".
+
+![Image showing the green status indicating how many devices have deployed this snapshot option in the menu.](./images/status-showing-the-target-snapshot.png){data-zoomable}
+*Image showing the green status indicating how many devices have deployed this snapshot option in the menu.*
+
+Once done you will be able to see the green mark in that snapshot showing on how much devices it is deployed on.
+
+## Common Mistakes to Avoid
+
+1. **Neglecting Regular Backups**: Don’t skip those snapshots! Regularly create backups, especially before making significant changes, moving devices through instances or applications, or disabling developer mode. Think of it as your safety net—ensuring you can always bounce back to a stable state if unexpected issues pop up.
+
+2. **Overlooking Auto-Snapshot Limits**: Did you know that your **auto snapshots** have limits? Be mindful of how many you can retain, as older **auto snapshots** will be automatically deleted. If you have important **auto snapshots**, save them locally to keep them safe. Remember, manually created snapshots have no limits, so take advantage of that!
```

Comment:
```suggestion
2. **Overlooking Auto-Snapshot Limits**: Did you know that your **auto snapshots** have limits? Be mindful of how many you can retain, as older **auto snapshots** will be automatically deleted. If you have important **auto snapshots**, either rename them to avoid automatic deletion or download and save them locally to keep them safe. Remember, manually created snapshots have no limits, so take advantage of that!
```